frontend/src/component/personalDashboard/createChartData.ts

```diff
@@ -0,0 +1,42 @@
+import type { IFeatureMetricsRaw } from 'interfaces/featureToggle';
```

Comment:
this is a copy paste of the feature metrics with colors swapped and total series removed